components/espocrm/actions/list-accounts/list-accounts.mjs

```diff
@@ -0,0 +1,28 @@
+import espoCrm from "../../espocrm.app.mjs";
+
+export default {
+  key: "espocrm-list-accounts",
+  name: "List Accounts",
+  description: "Retrieves a list of accounts from Espo CRM. [See the documentation](https://docs.espocrm.com/development/api/account/#list)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    espoCrm,
+  },
+  async run({ $ }) {
+    const items = this.espoCrm.paginate({
+      resourceFn: this.espoCrm.getAccounts,
+      args: {
+        $,
+      },
+    });
+    const accounts = [];
+    for await (const item of items) {
+      accounts.push(item);
+    }
```

Comment:
```suggestion
    const accounts = Object.values(items);
```

Reply:
Using `Object.values` doesn't work in this situation because `items` is coming from a generator function. We have to `await` the yield of each item from `*paginate`.